kubernetes/samples/scripts/domain-lifecycle/clusterStatus.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+set -eu
+set -o pipefail
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/helper.sh
+
+function usage() {
+cat<<EOF
+
+Usage:
+  $(basename $0) [-n DomainNS] [-d DomainUID] [-c clsterName] [-m kubernetesCli] [-h]
+
+  This script dumps a table of WKO domain status
+  replica information for each given cluster.
+
+  Omit '-n' or set it to "" to indicate all namespaces.
+  Omit '-d' or set it to "" to indicate all domains in the given namespace(s).
+  Omit '-c' or set it to "" to indicate all clusters in the given domain(s) and the given namespace(s).
+  Use '-m' to specify a Kubernetes command line interface (defaults to 'kubectl').
+  Use '-h' to get this help.
+
+Examples:
+  - Dump cluster 'c1' in uid 'myuid' in ns 'myns': $(basename $0) -n myns -d myuid -c c1
+
+  - Dump all clusters in uid 'myuid' in ns 'myns': $(basename $0) -n myns -d myuid
+                                                   -or-
+                                                   $(basename $0) -n myns -d "" -c ""
+   
+  - Dump all clusters and all uid in ns 'myns':    $(basename $0) -n myns
+                                                   -or-
+                                                   $(basename $0) -n myns -d ""
+
+  - Dump all uid across all ns:                    $(basename $0)
+
+Sample output:
+  WKO Cluster Status -n "" -d "" -c "":
+  namespace          domain          cluster    min  max  goal  current  ready
+  ---------          ------          -------    ---  ---  ----  -------  -----
+  sample-domain1-ns  sample-domain1  cluster-1  0    5    2     2        2
+
+EOF
+
+exit $1
+}
+
+# function to dump the domain cluster status in a table
+# $1=ns $2=uid, pass "" to mean "any"
+function clusterStatus() {
+  local __ns="${1:-}"
+  if [ -z "$__ns" ]; then
+    # an empty ns means check all namespaces
+    local __ns_filter="--all-namespaces=true"
+  else
+    local __ns_filter="-n $__ns"
+  fi
+
+  local __uid="${2:-}"
+  local __cluster_name="${3:-}"
+  local __kubernetes_cli="${4:-kubectl}"
+
+  validateKubernetesCliAvailable "$__kubernetes_cli"
+  failIfValidationErrors
```

Comment:
These two lines look like the only code grabbed from 'helper.sh'. To make this a completely stand-alone script, perhaps replace with 
```
  if ! [ -x "$(command -v ${__kubernetesCli})" ]; then
     echo "@@Error: Kuberenetes CLI '${kubernetesCli}' is not installed."
     exit 1
  fi
```